convert/convert.go

```diff
@@ -182,8 +182,10 @@ func ConvertModel(fsys fs.FS, ws io.WriteSeeker) error {
 
 	var conv ModelConverter
 	switch p.Architectures[0] {
-	case "LlamaForCausalLM", "MistralForCausalLM":
```

Comment:
@jmorganca @BruceMacD is this intentional? This effectively deregisters Mistral from the Ollama engine